data/quickstarts/deploy-model-rhoam-quickstart.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   annotations:
     opendatahub.io/categories: 'AI/Machine learning,API,Management,Model serving,Python'
 spec:
-  displayName: Getting Started with Red Hat OpenShift API Management
+  displayName: Securing a Deployed Model using Red Hat OpenShift API Management
```

Comment:
I like this -- do we need to mention it's a Python model? Curious what others think.

Other than that, just a minor suggestion to use sentence case (Securing a deployed model using ...).